### ISSUE TITLE
Mx Linux debian 10----WidevineCDM download error.

### ISSUE THREAD

**Issue by Rick654** [emilsvennesson/script.module.inputstreamhelper#433]
Hello developers.
System in use: MX linux debian
KOdi matrix 19

When I try to start some live content, I am asked to install widevine CDM, I click ok, but nothing happens, and I get the error you see in the screenshot.
How can I solve?

![116827836-7ff56000-ab9b-11eb-9f49-0279659c3476](https://user-images.githubusercontent.com/14813195/117074844-33915800-ad34-11eb-927c-c648355eb3b9.png)

Thanks

**Comment by mediaminister** [comment]
I need a full debug log to investigate your problem. Please read https://kodi.wiki/view/Log_file/Easy

**Comment by Rick654** [comment]
Thanks , mediaminister!
https://paste.kodi.tv/lirayugake.kodi
wish you did well!

**Comment by mediaminister** [comment]
I have a fix for your problem, can you test the following test package?
[script.module.inputstreamhelper-0.5.2+matrix.1-prev-dd63bce.zip](https://github.com/add-ons/plugin.video.vrt.nu/files/6436475/script.module.inputstreamhelper-0.5.2%2Bmatrix.1-prev-dd63bce.zip)

**Comment by Rick654** [comment]
Should I install as a normal zip file addon?

**Comment by Rick654** [comment]
I have updated, not solved

**Comment by mediaminister** [comment]
Yes, install as a normal zip file addon. I'll test more to make sure it works.

**Comment by mediaminister** [comment]
Please test this new test package. Install add-on as zip
[script.module.inputstreamhelper-0.5.2+matrix.1-prev-b2727ab.zip](https://github.com/emilsvennesson/script.module.inputstreamhelper/files/6437191/script.module.inputstreamhelper-0.5.2%2Bmatrix.1-prev-b2727ab.zip)

**Comment by Rick654** [comment]
Great, this worked.
Only now it asks me if I want to update widevine, but it never stops updating, and when I start Kodi, I get two errors:
1) remote server communication failed;
2) error tubed api.

I'm waiting for your news, thanks :-)

**Comment by mediaminister** [comment]
I guess this will work:
[script.module.inputstreamhelper-0.5.2+matrix.1-prev-4941c2f.zip](https://github.com/emilsvennesson/script.module.inputstreamhelper/files/6437530/script.module.inputstreamhelper-0.5.2%2Bmatrix.1-prev-4941c2f.zip)

**Comment by Rick654** [comment]
Excuse me. The two errors were caused by multiple instances of kodi running in the backgroud.
This latest script doesn't work.
Failed to play :-(

**Comment by mediaminister** [comment]
>  Some channels ask to update widevine: do I do this?

You can try or provide more info, a screenshot or debug log so I can investigate.

**Comment by Rick654** [comment]
![Istantanea_2021-05-07_12-02-32](https://user-images.githubusercontent.com/14813195/117434475-02c84300-af2d-11eb-8b49-ebb9142e39be.png)

Some channels, not all.
If you need the log let me know.
Thank you

**Comment by Testato** [comment]
@Rick654 
So is this problem completely solved for you also for LA7 plugin ?
https://github.com/luivit/plugin.video.rivedila7/issues/56
